helm-charts/verticadb-operator/README.md

```diff
@@ -23,5 +23,6 @@ This helm chart will install the operator and an admission controller webhook.
 | skipRoleAndRoleBindingCreation | Set this to true to force the helm chart to skip creation of any Roles and RoleBindings. This can only be used when the ServiceAccount already exists, so it expects serviceAccountNameOverride to have been used. <br><br> Use this option if you are installing the helm chart with k8s privileges that prevent you from creating Roles/RoleBindings. We provide the Roles and RoleBindings that the operator needs as an artifact of the GitHub release (see https://github.com/vertica/vertica-kubernetes/releases). | false |
 | resources.\* | The resource requirements for the operator pod. | <pre>limits:<br>  cpu: 100m<br>  memory: 750Mi<br>requests:<br>  cpu: 100m<br>  memory: 20Mi</pre> |
 | prometheus.expose | Controls exposing of the prometheus metrics endpoint.  Valid options are:<br><br>- **EnableWithAuthProxy**: A new service object will be created that exposes the metrics endpoint.  Access to the metrics are controlled by rbac rules using the proxy (see https://github.com/brancz/kube-rbac-proxy). The metrics endpoint will use the https scheme.<br><br>- **EnableWithoutAuth**: Like EnableWithAuthProxy, this will create a service object to expose the metrics endpoint.  However, there is no authority checking when using the endpoint.  Anyone who has network access to the endpoint (i.e. any pod in k8s) will be able to read the metrics.  The metrics endpoint will use the http scheme.<br><br>- **Disable**: Prometheus metrics are not exposed at all.  | EnableWithAuthProxy |
-| prometheus.createServiceMonitor | Set this to true if you want to create a ServiceMonitor.  This object is a CR provided by the prometheus operator to allow for easy service discovery.  If set to true, the prometheus operator must be installed before installing this chart.<br> See: https://github.com/prometheus-operator/prometheus-operator | false |
-| prometheus.createProxyRBAC | Set this to true if you want the helm install to create the rbac rules for accessing the metrics endpoint when it is protected by the rbac auth proxy.  If setting this to true, installation of this chart will fail if installing multiple times on the same system.  This happens because of the cluster scoped nature of the rbac rules. | false |
+| prometheus.createServiceMonitor | Set this to true if you want to create a ServiceMonitor.  This object is a CR provided by the prometheus operator to allow for easy service discovery.  If set to true, the prometheus operator must be installed before installing this chart.<br> See: https://github.com/prometheus-operator/prometheus-operator<br><br>*This parameter is deprecated and will be removed in a future release.* | false |
```

Comment:
Why are we deprecating this?

Reply:
It really isn't usable as-is. I tried to use it on our k8s-load-testing repo, but it needed to be modified to find the operator as a prometheus target. So, it really depends on how the prometheus operator is setup. It might be useful as a sample, but it originally was documented as a way to automatically include the operator in the prometheus stack.